include/vcpkg/base/message-data.inc.h

```diff
@@ -2559,6 +2568,11 @@ DECLARE_MESSAGE(RestoredPackagesFromAWS,
                 (msg::count, msg::elapsed),
                 "",
                 "Restored {count} package(s) from AWS in {elapsed}. Use --debug to see more details.")
+DECLARE_MESSAGE(RestoredPackagesFromAZUPKG,
+                (msg::count, msg::elapsed),
+                "",
+                "Restored {count} package(s) from Azure Universal Packages in {elapsed}. "
```

Comment:
```suggestion
                "Restored {count} package(s) from Universal Packages in {elapsed}. "
```

They are called "Universal Packages in Azure Artifacts" in https://learn.microsoft.com/en-us/azure/devops/artifacts/quickstarts/universal-packages?view=azure-devops&tabs=Windows -- I think the intended name does not have the 'Azure' prefix